docs/design/2022-05-13-refactor-transaction-module.md

```diff
@@ -0,0 +1,91 @@
+# Code structure of TiDB and Raft Module
```

Comment:
Rename the filename to be "2022-05-13-code-structure-of-tidb-and-raft-module.md"